collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -189,40 +189,24 @@ public static Bitmap getBitmapScaledToDisplay(File file, int screenHeight, int s
         BitmapFactory.Options options = new BitmapFactory.Options();
         options.inJustDecodeBounds = true;
         getBitmap(file.getAbsolutePath(), options);
```

Comment:
Since `if/else` block is removed and this instance of `options` is not being used to get the image's height/width, hence this call can be removed.